setup.py

```diff
@@ -55,6 +55,6 @@ def list_all_files(root_dir):
           'templates': list_all_files('templates'),
     },
     data_files=[
-         ('/etc/linchpin', ['linchpin_config.yml']),
+         ('linchpin_config.yml', ['linchpin_config.yml']),
```

Comment:
I don't believe this is the way you should do this. Instead, I'd suggest the following

```('', ['linchpin_config.yml'])

Additionally, the version should currently be 0.8.1, maybe? Definitely not 1.0. :)